src/OrbitBase/ExecutablePathLinuxTest.cpp

```diff
@@ -12,6 +12,11 @@
 #include "OrbitBase/Result.h"
 
 TEST(ExecutablePathLinux, GetExecutablePathWithPid) {
+  /* copybara:insert(executable is named differently)
+  GTEST_SKIP();
+  return;
```

Comment:
Doesn't `GTEST_SKIP();` already also `return`? Here and in the other tests.

Reply:
Uhhh. Good question. Let me check.

Reply:
Yes, works without the return. Thanks!

Reply:
Fixed!